loopchain/channel/channel_service.py

```diff
@@ -196,6 +196,7 @@ def cleanup(self):
         ChannelProperty().rest_target = kwargs.get('rest_target')
         ChannelProperty().radio_station_target = kwargs.get('rs_target')
         ChannelProperty().peer_id = kwargs.get('peer_id')
+        ChannelProperty().peer_address = ExternalAddress.fromhex_address(ChannelProperty().peer_id)
```

Comment:
How about integrating `peer_id` to `peer_address` and change usages of `ChannelProperty().peer_id` to `ChannelProperty().peer_address.hex_hx()`?

Reply:
More discussion is needed. It can be changed in other PR or Not.